CHANGELOG.md

```diff
@@ -15,6 +15,18 @@ pull request if there was one.
 
 ### Changed:
 
+- [Moved ResponseFormatType from Singleton enum to interface with enum imp](https://github.com/yahoo/fili/issues/711)
+    * Refactored ResponseFormatType to allow expansion via additional interface implementors
+    * Replaced equality based matching with 'accepts' model
```

Comment:
Just curious, what are the benefits of 'accepts' model?

Reply:
Enum equality is broken.  You can't compare enums with instances from other class hiearchies.  What we want is an equality-like operator but in which we can do a many to one comparison where 'CSV' can accept an instance of NamedExportReportFormat which implements ReportFormatType but also has state such as the name of the export header for that report.